src/loaders/ncbi_downloader/ncbi_downloader_helper.py

```diff
@@ -36,7 +36,10 @@ def _download_genome_file(
 
                 host.chdir(gene_dir)
                 dir_list = host.listdir(host.curdir)
-                gene_dir_name = [i for i in dir_list if i.startswith(gene_id)][0]
+                gene_dir_list = [i for i in dir_list if i.startswith(gene_id)]
+                if len(gene_dir_list) > 1:
```

Comment:
Might as well just do `== 1` and make sure there's at least 1 dir as well

Reply:
Updated!